include/sacn/merge.h

```diff
@@ -0,0 +1,162 @@
+/******************************************************************************
+ * Copyright 2020 ETC Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************
+ * This file is a part of sACN. For more information, go to:
+ * https://github.com/ETCLabs/sACN
+ *****************************************************************************/
+
+/*!
+ * \file sacn/merge.h
+ * \brief DMX Merge API definitions
+ *
+ * Functions and definitions for the \ref dmx_merger "DMX Merger API" are contained in this
+ * header.
+ */
+
+#ifndef DMX_MERGER_H_
+#define DMX_MERGER_H_
```

Comment:
I've adopted a convention for these projects where the header guard matches the file name (scoped with subdirectories starting with the subdirectory matching the name of the library).

So, this should be:
```
#ifndef SACN_MERGE_H_
#define SACN_MERGE_H_
```